orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -143,11 +144,20 @@ def _replicate_for_data_parallel(self):
         which does not need model replication and is also recommended by torch to use instead.
         """
 
+        policy = os.getenv("ORTMODULE_FALLBACK_POLICY")
+        if policy and policy is _FallbackPolicy.FALLBACK_DISABLE:
```

Comment:
is this the right way to do fallback check? 

This is the one I see from existing cases. 
    if os.getenv("ORTMODULE_FALLBACK_POLICY") == _FallbackPolicy.FALLBACK_DISABLE.name:
        return False

    if not policy:
        policy = _FallbackPolicy.FALLBACK_DISABLE

Reply:
added `.name` for string compare